api/proto/realtime/realtime.proto

```diff
@@ -84,8 +85,10 @@ enum MessageType {
   SERVER_MESSAGE = 24;
   READY_FOR_GAME = 25;
   LAG_MEASUREMENT = 26;
-  TOURNAMENT_GAME_ENDED_EVENT = 27;
+  TOURNAMENT_MESSAGE = 27;
```

Comment:
don't reuse this message - can you make it a new one? we can obsolete the old one later (mark it obsolete). reusing messages in pb can really result in issues.

Reply:
Unreused.